lib/rules/linebreaks/index.js

```diff
@@ -60,9 +60,7 @@ const rule = (primary, _secondaryOptions, context) => {
 
 			const lines = root.source.input.css.split('\n');
 
-			for (let i = 0; i < lines.length; i++) {
-				let line = lines[i];
-
+			for (let [i, line] of lines.entries()) {
```

Comment:
[note] [`.entries()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries) makes a traditional `for` loop much readable.